app/test_main.py

```diff
@@ -1,3 +1,67 @@
-from app.main import get_human_age
+import pytest
 
-# write your code here
+from app.main import get_human_age, convert_to_human
+
+
+@pytest.mark.parametrize(
+    "testing_cat_age, testing_dog_age, expected_result",
+    [
+        pytest.param(
+            15, 15, [1, 1],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
+            24, 24, [2, 2],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
+            100, 100, [21, 17],
+            id="result_should_be_correct"
+        ),
+        pytest.param(
```

Comment:
I think it's better  to move this test to the beginning and put cases in the sort order